packages/vx-axis/src/axis/AxisRadial.js

```diff
@@ -0,0 +1,287 @@
+   */
+  tickFormat: PropTypes.func,
+  /**
+   * A function that returns props for a given tick value and index.
+   */
+  tickLabelProps: PropTypes.func,
+  /**
+   * The length of the tick lines.
+   */
+  tickLength: PropTypes.number,
+  /**
+   * The color for the tick's stroke value.
+   */
+  tickStroke: PropTypes.string,
+  /**
+   * A custom SVG transform value to be applied to each tick group.
+   */
+  tickTransform: PropTypes.string,
+  /**
+   * An array of values that determine the number and values of the ticks.
+   * Falls back to `scale.ticks()` or `.domain()`.
+   */
+  tickValues: PropTypes.array,
+  /**
+   * For more control over tick label rendering, pass a function
+   */
+  tickComponent: PropTypes.func,
+  /**
+   * A top pixel offset applied to the entire axis.
+   */
+  top: PropTypes.number,
+  /**
+   * For more control over rendering or to add event handlers to datum, pass a function as children.
+   */
+  children: PropTypes.func
+};
+
+export default function AxisRadial({
+  children,
+  axisClassName,
+  axisLineClassName,
+  hideAxisLine = false,
+  hideTicks = false,
+  hideZero = false,
+  label = '',
+  labelClassName,
+  labelOffset = 14,
+  labelOrientation = 'top',
+  labelProps = {
+    fontFamily: 'Arial',
+    fontSize: 12,
+    fontWeight: 'bold',
+    fill: 'black'
+  },
+  left = 0,
+  numTicks = 10,
+  radius,
+  scale,
+  stroke = 'black',
+  strokeWidth = 1,
+  strokeDasharray,
+  tickClassName,
+  tickFormat,
+  tickLabelProps = (tickValue, index) => ({
+    fontFamily: 'Arial',
+    fontSize: 10,
+    fill: 'black'
+  }),
+  tickLength = 8,
+  tickStroke = 'black',
+  tickTransform,
+  tickValues,
+  tickComponent,
+  top = 0
+}) {
+  let values = scale.ticks ? scale.ticks(numTicks) : scale.domain();
+  if (tickValues) values = tickValues;
+  let format = scale.tickFormat ? scale.tickFormat() : identity;
+  if (tickFormat) format = tickFormat;
+
+  let tickLabelFontSize = 10; // track the max tick label size to compute label offset
+
+  const ticks = values
+    .map((val, index) => {
+      if (hideZero && val === 0) return null;
+
+      const angle = scale(val) - Math.PI / 2;
+      const tickFromPoint = new Point(polarToCartesian({ angle, radius }));
+      const tickToPoint = new Point(polarToCartesian({ angle, radius: radius + tickLength }));
+
+      const tickOrientationProps = radialTickLabelTransform(tickFromPoint, tickToPoint);
+      const tickLabelPropsObj = { ...tickOrientationProps, ...tickLabelProps(val, index) };
+      tickLabelFontSize = Math.max(tickLabelFontSize, tickLabelPropsObj.fontSize || 0);
+
+      return tickLabelPropsObj;
+    })
+    .filter(defined => !!defined);
+
+  const axisLineData = scale.ticks(50);
+  const angle = d => scale(d);
+
+  if (children) {
+    return (
+      <Group className={cx('vx-axis-radial', axisClassName)} top={top} left={left}>
```

Comment:
all classes are the same as other `Axis*` components except this top-level one which should be sufficient to distinguish from other (linear) axes on the page.